lib/events/events_handler.dart

```diff
@@ -105,12 +105,6 @@ class EventsHandler {
         }
         break;
 
-      case "onNoticeClickViewSPIPurposes":
```

Comment:
these events are deprecated but not removed. Do you think we can remove them now? it would require a major version (breaking changes)

Reply:
The events are still present in dart code. What we are removing here, is the bridge between native events and dart events, which should not make any difference as the events are not triggered anymore from native.

Keeping them would require to add an `ignore` tag to avoid Lint failure, imo it is easier to remove them now.